### ISSUE TITLE
how can i use schtaskscreate command ?

### ISSUE THREAD

**Issue by breaking153** [trustedsec/CS-Remote-OPs-BOF#16]
it looks like whatever i input,it returns xml malform error 
![图片](https://user-images.githubusercontent.com/49428332/233285122-a4837477-62f5-4cd2-93a9-33ba5c0b8d08.png)
![图片](https://user-images.githubusercontent.com/49428332/233285395-c161ced4-c72e-4d06-b449-e4c233c32efd.png)

xml content,the xml i use is copy from tasks dir btw.:
```
<?xml version="1.0" encoding="UTF-16"?>
<Task version="1.2" xmlns="http://schemas.microsoft.com/windows/2004/02/mit/task">
  <RegistrationInfo>
    <Date>2023-04-20T14:40:18</Date>
    <Author>Administrator</Author>
  </RegistrationInfo>
  <Triggers>
    <TimeTrigger>
      <Repetition>
        <Interval>PT30M</Interval>
        <StopAtDurationEnd>false</StopAtDurationEnd>
      </Repetition>
      <StartBoundary>2023-04-20T14:40:00</StartBoundary>
      <Enabled>true</Enabled>
    </TimeTrigger>
  </Triggers>
  <Settings>
    <MultipleInstancesPolicy>IgnoreNew</MultipleInstancesPolicy>
    <DisallowStartIfOnBatteries>true</DisallowStartIfOnBatteries>
    <StopIfGoingOnBatteries>true</StopIfGoingOnBatteries>
    <AllowHardTerminate>true</AllowHardTerminate>
    <StartWhenAvailable>false</StartWhenAvailable>
    <RunOnlyIfNetworkAvailable>false</RunOnlyIfNetworkAvailable>
    <IdleSettings>
      <Duration>PT10M</Duration>
      <WaitTimeout>PT1H</WaitTimeout>
      <StopOnIdleEnd>true</StopOnIdleEnd>
      <RestartOnIdle>false</RestartOnIdle>
    </IdleSettings>
    <AllowStartOnDemand>true</AllowStartOnDemand>
    <Enabled>true</Enabled>
    <Hidden>false</Hidden>
    <RunOnlyIfIdle>false</RunOnlyIfIdle>
    <WakeToRun>false</WakeToRun>
    <ExecutionTimeLimit>PT72H</ExecutionTimeLimit>
    <Priority>7</Priority>
  </Settings>
  <Actions Context="Author">
    <Exec>
      <Command>C:\Users\administrator\desktop\calc.exe</Command>
    </Exec>
  </Actions>
  <Principals>
    <Principal id="Author">
      <UserId>WIN-MHCC9K07ED3\Administrator</UserId>
      <LogonType>InteractiveToken</LogonType>
      <RunLevel>LeastPrivilege</RunLevel>
    </Principal>
  </Principals>
</Task>
```

**Comment by helloyw** [comment]
>师傅请问你解决掉这个问题了吗？

**Comment by freefirex** [comment]
I was able to look into this today.
I recreated your error when I saved the XML you provided using UTF-16 as the file encoding.
When I re-saved the file using UTF-8 encoding the schtaskscreate command would then accept it.

I did not change what the file said it used for encoding at the top, and I'm unsure why the API I'm using appears to not respect it.  

Let me know if saving the xml using UTF-8 encoding fixes it for you as well.
